dataclients/kubernetes/testdata/ingressV1/ingress-data/ing-with-duplicate-host-different-service.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  namespace: foo
+  name: qux
+spec:
+  defaultBackend:
+    service:
+      name: bar
+      port:
+        number: 1234
+  rules:
+  - host: www.example.org
+    http:
+      paths:
+      - path: "/"
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: bar
+            port:
+              name: baz
+  - host: www.example.org
+    http:
+      paths:
+      - path: "/"
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: bar
+            port:
+              name: baz
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: foo
+  name: bar
+spec:
+  clusterIP: 10.3.190.97
+  ports:
+  - name: baz
+    port: 8181
+    protocol: TCP
+    targetPort: 8080
+  - name: qux
+    port: 1234
+    protocol: TCP
+    targetPort: 2134
+  selector:
+    application: myapp
+  type: ClusterIP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: foo
+  name: bar2
+spec:
+  clusterIP: 10.3.190.97
+  ports:
+  - name: baz2
+    port: 9191
+    protocol: TCP
+    targetPort: 8081
+  selector:
+    application: myapp
+  type: ClusterIP
+---
+apiVersion: v1
+kind: Endpoints
+metadata:
+  labels:
+    application: myapp
+  namespace: foo
+  name: bar
+subsets:
+- addresses:
+  - ip: 10.2.9.103
+  - ip: 10.2.9.104
+  ports:
+  - name: baz
+    port: 8080
+    protocol: TCP
+  - name: qux
+    port: 2134
+    protocol: TCP
+---
+apiVersion: v1
+kind: Endpoints
+metadata:
+  labels:
+    application: myapp
+  namespace: foo
+  name: bar2
+subsets:
+- addresses:
+  - ip: 10.2.9.103
+  - ip: 10.2.9.104
```

Comment:
Lets use different fake IPs for different endpoints

Reply:
Make it more realistic.
Different services normally have different selectors and thus select different pods.
Route backend endpoint list depends on the values of these IPs - with the same IPs routes could become equal while they should not.